src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java

```diff
@@ -62,6 +62,9 @@ public void testRemoveFromCourseAction(){
         
         detailsPage.clickRemoveInstructorFromCourse(courseId)
             .verifyStatus(Const.StatusMessages.INSTRUCTOR_REMOVED_FROM_COURSE);
+        
+        detailsPage.verifyHtml("/adminAccountDetailsRemoveInstructor.html");
```

Comment:
Maybe a html comparison here is a bit superfluous. Is there anything in particular you would like to test here?